src/D2L.CodeStyle.Analyzers/CommonFixes/AddAttributeCodeFix.cs

```diff
@@ -108,11 +118,11 @@ string usingNs
 			return root.WithUsings( root.Usings.Add( usingDirective ) );
 
 			bool IsTheNecessaryUsingDirective( UsingDirectiveSyntax u ) {
-				if ( u.StaticKeyword == null && usingStatic ) {
+				if ( u.StaticKeyword.Kind() == SyntaxKind.None && usingStatic ) {

```

Comment:
Old checks were actually busted because StaticKeyword is a struct and never null :P
It worked fine for `using static D2L.CodeStyle.Annotations.Objects` but nothing else.